.github/workflows/test.yml

```diff
@@ -129,6 +127,14 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set tag (staging)
+        # Set "staging" as Docker tag if the event is a push to main branch
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+        run: echo "DOCKER_TAG=staging" >> $GITHUB_ENV
+      - name: Set tag (release)
+        # Set the tag versions as Docker tag if it is a release event
+        if: github.event_name == 'release' && github.event.action == 'released'
+        run: echo "DOCKER_TAG=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
```

Comment:
@jpalvarezl I think it's not possible to have multiple run conditions on a single `step`. See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsif

Reply:
That's unfortunate! but this seems like the best we can do with what we have.